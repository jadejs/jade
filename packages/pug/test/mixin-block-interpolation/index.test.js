const pug = require('../../');

test('mixin block tag interpolation', () => {
    const output = pug.renderFile(
        __dirname + '/index.pug',
        {pretty: true}
    );
    expect(output).toMatchSnapshot();
});