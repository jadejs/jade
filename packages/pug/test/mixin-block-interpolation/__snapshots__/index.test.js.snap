// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mixin block tag interpolation 1`] = `
"<span>proof that 
  <block></block></span>
<p>another paragraph</p>"
`;
